packages/react-examples/src/react/DetailsList/DetailsList.ColumnResize.Example.tsx

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import { IColumn } from 'office-ui-fabric-react/lib/DetailsList';
```

Comment:
These should all be `@fluentui/react/lib/whatever` (also it would be preferable to combine the DetailsList imports into one)

Reply:
@ecraig12345 Any reason not to just combine them all into a single import statement `from '@fluentui/react'`?

Reply:
Done.